app/javascript/packs/application.js

```diff
@@ -30,16 +30,16 @@ import "bootstrap";
 
 // Internal imports, e.g:
 // import { initSelect2 } from '../components/init_select2';
-// import { toggleRequired } from 'pluggins/checkbox';
+// import { toggleRequired } from '../pluggins/checkbox';
```

Comment:
Not sure we need the ".." before the name of the folder here, because there are on the same big folder, no??